src/styles/scss/shared/headers/app-header.scss

```diff
@@ -0,0 +1,29 @@
+@import './header';
```

Comment:
preemptively doing this for when I change everything to load-only-what-you-need approach to scss.